book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    Rental.new(date, self, person)
+  end
+end
```

Comment:
- [ ] All your code structures are clean 🔥. Keep rocking!!

Reply:
Thank you!